dashboard/src/dashboard.py

```diff
@@ -313,6 +315,10 @@ def main():
         delete_work_files(repository)
         update_overall_status(results, repository)
 
+        results.ci_jobs[repository] = {}
```

Comment:
Or you can use [defaultdict](https://docs.python.org/3.3/library/collections.html#defaultdict-examples)

```
In [1]: from collections import defaultdict

In [2]: ci_jobs = defaultdict(dict)

In [3]: ci_jobs['xxx']['yyy'] = 1

In [4]: print(ci_jobs)
defaultdict(<class 'dict'>, {'xxx': {'yyy': 1}})

In [5]: ci_jobs['xxx']
Out[5]: {'yyy': 1}
```